### PR TITLE
gh-81652 add MAP_ALIGNED_SUPER FreeBSD and MAP_CONCEAL OpenBSD constants

### PR DESCRIPTION
**Pull request by yeojin-dev**
* Issue: gh-81652

### CODE REVIEW

File: Misc/NEWS.d/next/Library/2023-02-23-20-39-52.gh-issue-81652.Vxz0Mr.rst [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Add MAP_ALIGNED_SUPER FreeBSD and MAP_CONCEAL OpenBSD constants to mmap module
+
```
Comment (corona10) [inline Misc/NEWS.d/next/Library/2023-02-23-20-39-52.gh-issue-81652.Vxz0Mr.rst]:
```suggestion
Add :data:`mmap.MAP_ALIGNED_SUPER` FreeBSD and :data:`mmap.MAP_CONCEAL 
OpenBSD constants to :mod:`mmap`. Patch by Yeojin Kim.
```

File: Doc/library/mmap.rst [hunk 2]
```diff
@@ -378,3 +380,7 @@ MAP_* Constants
 
     .. versionadded:: 3.11
        Added MAP_STACK constant.
+
+    .. versionadded:: 3.12
+       Added MAP_ALIGNED_SUPER constant.
+       Added MAP_CONCEAL constant.
```
Comment (corona10) [inline Doc/library/mmap.rst]:
```suggestion
       Added :data:`MAP_ALIGNED_SUPER` constant.
       Added :data:`MAP_CONCEAL` constant.
```

File: Doc/library/mmap.rst [hunk 3]
```diff
@@ -370,6 +370,8 @@ MAP_* Constants
           MAP_ANONYMOUS
           MAP_POPULATE
           MAP_STACK
+          MAP_ALIGNED_SUPER
+          MAP_CONCEAL
 
     These are the various flags that can be passed to :meth:`mmap.mmap`. Note that some options might not be present on some systems.
```
Comment (corona10) [inline Doc/library/mmap.rst]: Please describe that MAP_ALIGNED_SUPER and MAP_CONCEAL are only available at OpenBSD.

**Review by corona10 — CHANGES_REQUESTED** [review]

File: Misc/NEWS.d/next/Library/2023-02-23-20-39-52.gh-issue-81652.Vxz0Mr.rst [hunk 4]
```diff
@@ -0,0 +1,2 @@
+Add :data:`mmap.MAP_ALIGNED_SUPER` FreeBSD and :data:`mmap.MAP_CONCEAL
```
Comment (corona10) [inline Misc/NEWS.d/next/Library/2023-02-23-20-39-52.gh-issue-81652.Vxz0Mr.rst]:
```suggestion
Add :data:`mmap.MAP_ALIGNED_SUPER` FreeBSD and :data:`mmap.MAP_CONCEAL`
```

**Comment by yeojin-dev** [comment]
I have made the requested changes; please review again

**Review by corona10 — APPROVED** [review]
lgtm

**Pull request merged**